Tester.js

```diff
@@ -239,6 +181,17 @@ Tester.prototype = {
     next();
   },
 
+  modifyAndAssertJSON: function(custom_assertion) {
```

Comment:
This is for applying an assertion in the JSON with a custom callback for modifying the JSON before the comparison takes place, so to account for dynamic response values, like ID fields, for example.

    this.Then(/^the response should match the following, taking into account that "([^"]*)" are dynamic fields:$/, function(fields, exp_json, callback) {
        fields = fields.split(',');
        this.api.modifyAndAssertJSON(function(actual_json, call){
                for (i=0; i < fields.length; i++) {
                    var field_name = fields[i];
                    actual_json[field_name] = "";
                }
                call(exp_json, actual_json);
            })
            .then(callback);
    });